src/Components/DeathReport/DeathReport.tsx

```diff
@@ -111,7 +112,10 @@ export default function PrintDeathReport(props: { id: string }) {
         const patientComorbidities = getPatientComorbidities(res.data);
         const data = {
           ...res.data,
-          age: patientAgeInYears(res.data!),
+          age:
+            patientAgeInYears(res.data!) == 0
+              ? formatPatientAge(res.data!, true)
+              : patientAgeInYears(res.data!),
```

Comment:
the `formatPatientAge` function takes care of all edge cases right? why use `patientAgeInYears`? Isn't it redundant?

Reply:
@rithviknishad Yes. But If the age is greater than 1, it will add "yr" if we use formatPatientAge(res.data! ,true) . If this behavior is fine with you, I'll go ahead and update the commit. Previously, this detail wasn't shown. Let me know if you'd like to proceed with this.